File: Shefaa/Models/Receptionist.cs
namespace Shefaa.Models
{
    public enum ReceptionistStatus
    {
        Pending,
        Approved,
        Rejected,
        Suspended
    }
    public class Receptionist
    {
        public Guid ReceptionistId { get; set; }
        public Guid UserId { get; set; }
        public Guid BranchId { get; set; }
        public ApplicationUser User { get; set; }
        public Branch Branch { get; set; }
        [Required]
        public ReceptionistStatus Status { get; set; } = ReceptionistStatus.Pending;
    }
}

File: Frontend/src/features/receptionist/types/receptionist.ts
export interface ReceptionistResponse {
    receptionistId: string;
    userId: string;
    firstName: string;
    lastName: string;
    email: string;
    phoneNumbers: string[];
    branchName: string;
    profileImageUrl?: string;
    status: number; // 0 = Pending, 1 = Approved, 2 = Rejected, 3 = Suspended (Assuming Enum)
}

export interface ApiResponse<T> {
    isSuccess: boolean;
    message: string;
    data?: T;
    errors?: string[];
}
